sdk/src/types/block/output/account.rs

```diff
@@ -504,6 +542,33 @@ impl AccountOutput {
 
         Ok(())
     }
+
+    fn stored_len(&self) -> usize {
+        // Type
+        size_of::<u8>()
+            // Amount
+            + size_of::<u64>()
+            // Mana
+            + size_of::<u64>()
+            // Native Tokens
+            + size_of::<u8>()
+            + self.native_tokens.iter().map(|nt| nt.packed_len()).sum::<usize>()
+            // AccountId
+            + AccountId::LENGTH
+            // State Index
+            + size_of::<u32>()
```

Comment:
Doesn't exist anymore

Reply:
I think we should remove these methods anyway, it's just packed_len but error prone

Reply:
So you would use `self.{field}.packed_len()` everywhere?

Reply:
I'm just wondering what's the point of `stored_len` to start with